aspnetcore/grpc/aspnetcore.md

```diff
@@ -61,66 +61,36 @@ ASP.NET Core middlewares and features share the routing pipeline, therefore an a
 Kestrel gRPC endpoints:
 
 * Require HTTP/2.
-* Should be secured with HTTPS.
+* Should be secured with [Transport Layer Security (TLS)](https://tools.ietf.org/html/rfc5246).
 
 #### HTTP/2
 
 gRPC requires HTTP/2. gRPC for ASP.NET Core validates [HttpRequest.Protocol](xref:Microsoft.AspNetCore.Http.HttpRequest.Protocol*) is `HTTP/2`.
 
 Kestrel [supports HTTP/2](xref:fundamentals/servers/kestrel#http2-support) on most modern operating systems. Kestrel endpoints are configured to support HTTP/1.1 and HTTP/2 connections by default.
 
-#### HTTPS
+#### TLS
 
-Kestrel endpoints used for gRPC should be secured with HTTPS. In development, an HTTPS endpoint is automatically created at `https://localhost:5001` when the ASP.NET Core development certificate is present. No configuration is required.
+Kestrel endpoints used for gRPC should be secured with TLS. In development, an endpoint secured with TLS is automatically created at `https://localhost:5001` when the ASP.NET Core development certificate is present. No configuration is required. You can double check a Kestrel endpoint is secured by looking at the address. A `https` prefix indicates that the endpoint is secure.
```

Comment:
Kestrel endpoints used for gRPC should be secured with TLS. In development, an endpoint secured with TLS is automatically created at `https://localhost:5001` when the ASP.NET Core development certificate is present. No configuration is required.  An `https` prefix verifies the Kestrel endpoint is using TLS.

Reply:
I don't think @blowdart  would agree HTTPS guarantees the endpoint is secure. It only guarantees it's using TLS.

Reply:
Fair. Updated